addons/test_mail_full/static/tests/tours/message_actions_tour.js

```diff
@@ -12,3 +13,75 @@ registry.category("web_tour.tours").add("star_message_tour", {
         },
     ],
 });
+
+registry.category("web_tour.tours").add("message_actions_tour", {
+    steps: () => [
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Thread .o-mail-Message",
+            run: async function () {
+                await contains(".o-mail-Thread .o-mail-Message", {
+                    count: 1,
+                    target: document.querySelector("#chatterRoot").shadowRoot,
+                });
+            },
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Composer-input",
+            run: "edit New message",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Composer button:contains(Send):enabled",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Thread .o-mail-Message",
+            run: async function () {
+                await contains(".o-mail-Thread .o-mail-Message", {
+                    count: 2,
+                    target: document.querySelector("#chatterRoot").shadowRoot,
+                });
+            },
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message:contains(New message)",
+            run: "hover && click #chatterRoot:shadow button[title='Add a Reaction']",
+        },
```

Comment:
We might need to wait for the message to be persistent?

Reply:
Temporary message is shown first, actions are not available on it.

Test should have `[data-persistent]` to avoid race conditions.